morecantile/models.py

```diff
@@ -121,6 +127,7 @@ class TileMatrixSet(BaseModel):
     wellKnownScaleSet: Optional[AnyHttpUrl] = None
     boundingBox: Optional[TMSBoundingBox]
     tileMatrix: List[TileMatrix]
+    is_quadtree: bool = False
```

Comment:
I wonder if we should use `PrivateAttr` https://pydantic-docs.helpmanual.io/usage/models/#private-model-attributes for this. `is_quadtree` is not something defined in the TMS spec so I think it would be better if we make it somehow `private` cc @adrian-knauer @geospatial-jeff 

FYI: I didn't know about PrivateAttr until today, so I guess this is new to Pydantic

Reply:
This makes a lot of sense to me! I wasn't aware of `PrivateAttr` as well.